infra/base-images/base-runner/run_fuzzer

```diff
@@ -103,17 +115,18 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
   export UBSAN_OPTIONS="$UBSAN_OPTIONS:symbolize=0"
   export AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1
   export AFL_SKIP_CPUFREQ=1
-  export AFL_TRY_AFFINITY=1
+  export AFL_NO_AFFINITY=1
```

Comment:
I think these changes are accidental right? Maybe you didn't merge properly?
Can you try again.

Reply:
Mhh, sure? The latest commit on google:master has `AFL_NO_AFFINITY`.

Reply:
I think it has `AFL_TRY_AFFINITY=1`
See https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-runner/run_fuzzer#L106